dGame/EntityManager.cpp

```diff
@@ -37,7 +37,10 @@ std::vector<LWOMAPID> EntityManager::m_GhostingExcludedZones = {
 	1250,
 	1251,
 	1350,
-	1450
+	1450,
+
+	// Zorillo Plaza
+	3200
```

Comment:
This should not be on DLU main

Reply:
Didn't even mean to push that as it was a bodge change.  Fixes